css/utilities.css

```diff
@@ -0,0 +1,27 @@
+* {
+    box-sizing: border-box;
+}
+
+.disp-block {
+    display: block;
+}
+
+.disp-inline {
+    display: inline;
+}
+
+.disp-flex {
+    display: flex;
+}
+
+.flx-spc-btw {
```

Comment:
class words consistency can be fixed. Some place it's 'flex' and here 'flx' , so that can be made consistent.